.github/workflows/ci.yaml

```diff
@@ -0,0 +1,77 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+jobs:
+  checks:
+    name: Code style
+    runs-on: ubuntu-latest
```

Comment:
Is is possible to pin this ? Otherwise is there the potential for changes in ubuntu base image to break the builds

Reply:
Pinned.